src/lang/korean.lng

```diff
@@ -0,0 +1,85 @@
+##grflangid 0x3A
```

Comment:
https://github.com/OpenTTD/OpenTTD/blob/35dbd53682bd8e14fd0726480e47587b27a0d9ae/src/lang/korean.txt#L10

OpenTTD's translation file for `korean.txt` uses `##grflangid 0x3a` (lowercase instead of capital).

Reply:
Fixed, thx